packages/ember-htmlbars/lib/helpers/-legacy-each-with-controller.js

```diff
@@ -17,7 +17,7 @@ export default function legacyEachWithControllerHelper(params, hash, blocks) {
     var self;
 
     if (blocks.template.arity === 0) {
-      Ember.deprecate(deprecation);
+      Ember.deprecate(deprecation, false, { id: 'ember-htmlbars.each-with-controller-helper', until: '3.0.0' });
```

Comment:
Let's update to 2.4.0. This is only supported in 2.x with the legacy-views addon which is only intended to be supported until 2.4.0.